SimpleMap/LocationDataManager.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  ViewController.swift
+//  SimpleMap
+//
+//  Created by Artem Eliseev on 05/07/2023.
+//
+
+import Foundation
+import CoreLocation
+import UIKit
+import MapKit
+
+
+class LocationDataManager : NSObject, ObservableObject, CLLocationManagerDelegate {
```

Comment:
if a class is not going to be inherited from, which is most likely the case here then define the class as `final class LocationDataManager... ` with the final keyword